blockly-core/core/ui/block.js

```diff
@@ -1077,12 +1093,12 @@ Blockly.Block.prototype.onMouseMove_ = function(e) {
     var x = this.startDragX + dx;
     var y = this.startDragY + dy;
     this.svg_.getRootElement().setAttribute('transform',
-        'translate(' + x + ', ' + y + ')');
+      'translate(' + x + ', ' + y + ')');
```

Comment:
Did your editor settings update to two-space line continuation indent or something?